yaml-cpp.sh

```
@@ -1,25 +1,21 @@
 package: yaml-cpp
 version: "%(tag_basename)s"
-tag: yaml-cpp-0.6.3
+tag: 0.8.0
 source: https://github.com/jbeder/yaml-cpp
-requires:
-  - boost
 build_requires:
   - CMake
   - alibuild-recipe-tools
 prefer_system: (?!slc5)
 prefer_system_check: |
```

Shellcheck (0.11.0)

[error] Expected a { to open the function definition.

(SC1064)